restaurant/scripts.js

```diff
@@ -0,0 +1,431 @@
+async function getData() {
+  var response = await fetch("./menu.json");
+  var data = await response.json();
+  console.log(data);
```

Comment:
Please remove all console logs , 
We must never log api response as it may contain private information of the user such as name ,password etc